### ISSUE TITLE
pandas warning of deprecated `S`

### ISSUE THREAD

**Issue by braingram** [BAMWelDX/weldx#909]
In the asdf CI (which tests weldx as part of it's downstream) I'm seeing the following warning message:
```
E   FutureWarning: 'S' is deprecated and will be removed in a future version, please use 's' instead.
```
See here for context:
https://github.com/asdf-format/asdf/actions/runs/7717012615/job/21035144518?pr=1745#step:10:1148

This might be coming from a use of a deprecated [period alias](https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#period-aliases) but I can't quite pin down where this is coming from.

Is this a known issue? If so, do you recommend adding a warning filter to the asdf downstream job that tests weldx? Thanks!

**Comment by CagtayFabry** [comment]
Hi @braingram thank you for the heads-up

I guess this comes from some test cases that generate pandas time indexes. Unfortunately I am not sure when I have time to look into it further, it might be a while

**Comment by braingram** [comment]
Thanks for the response. I'll let you know if I find out anything more.

**Comment by marscher** [comment]
Have these aliases already been removed in Pandas-2.2?

**Comment by braingram** [comment]
I'm not sure (I'm honestly not that familiar with pandas).

A recent run of the asdf tests in weldx by the asdf downstream CI passed (with the warnings mentioned in this issue):
https://github.com/asdf-format/asdf/actions/runs/8137112282/job/22342599707#step:10:404
using pandas 2.2 so I think that means the aliases have not yet been removed.

**Comment by marscher** [comment]
> using pandas 2.2 so I think that means the aliases have not yet been removed.

you're right, they just have been deprecated. I general I really appreciate these kind of early failure notices! Thanks a lot.

@CagtayFabry Maybe we should have a workflow which turns deprecationwarnings and futurewarnings into exceptions and let it run weekly?
